examples/camera/first_person_view_model.rs

```diff
@@ -219,18 +221,33 @@ fn spawn_text(mut commands: Commands) {
 }
 
 fn move_player(
+    time: Res<Time>,
     accumulated_mouse_motion: Res<AccumulatedMouseMotion>,
     mut player: Query<&mut Transform, With<Player>>,
 ) {
     let mut transform = player.single_mut();
     let delta = accumulated_mouse_motion.delta;
+    let dt = time.delta_seconds();
+    // The factors are just arbitrary mouse sensitivity values.
```

Comment:
Pull these out into constants please :)

Reply:
I disagree. Pulling them into `MOUSE_SENSITIVITY_X` and `MOUSE_SENSITIVITY_Y`
- makes them less nicely grouped than now, and
- is semantically wrong. `PI` is a constant, an aspect ratio is a constant. Mouse sensitivity is a value that should be a configurable variable for accessibility.

As a compromise, I could extract them into a component if you want. Or add a component mentioning that this should be configurable.

Reply:
Component is nice: it makes it easy for the values to be twiddled via inspectors.

Reply:
[`3e372d1` (#15109)](https://github.com/bevyengine/bevy/pull/15109/commits/3e372d1362bdd8a519dde4bcabbc4983122225b6)